news/2025/2025-04-15-the-followpoint-azer-the-statistics-breaking-magician.md

```diff
@@ -0,0 +1,305 @@
+---
+layout: post
+title: "The Followpoint: Azer, the Statistics-Breaking Magician"
+date: 2025-04-15 17:00:00 +0000
+series: The Followpoint
+---
+
+You hear that? That's a sound of a new interview with someone who's a renowned player AND a World Cup organiser too — Azer!
+
+![](/wiki/shared/news/2025-04-15-the-followpoint-azer-the-statistics-breaking-magician/banner.jpg)
+
+~~Azer isn't so great? Are you kidding me? When was the last time you saw a player with such aim ability and movement with a tablet? Alex puts the game on another level, and we will be blessed if we ever see a player with his skill and passion for the game again. Cookiezi breaks records. Rafis breaks records. Azer breaks the rules. You can keep your statistics. I prefer the magic.~~
+
+On a serious note though, [Azer](https://osu.ppy.sh/users/2155578) is someone you most likely heard of if you are interested in osu! World Cups and especially if you are an older player. He's contributed a lot to the tournament scene and to World Cups specfiically, earning multiple [badges](/wiki/Community/Profile_badge) for that, ranging from pooling to organising the entire thing! Not to mention that he also received a [Community Contributor badge](/wiki/People/Community_Contributors) for all his work!
```

Comment:
```suggestion
On a serious note though, [Azer](https://osu.ppy.sh/users/2155578) is someone you have likely heard of if you are interested in osu! World Cups, and especially if you are an older player. He's contributed a lot to the tournament scene, earning multiple [badges](/wiki/Community/Profile_badge) in his time playing. He has also contributed heavily to the osu! World Cups, ranging from pooling to organising the entire thing! Not to mention that he also received a [Community Contributor badge](/wiki/People/Community_Contributors) for his dedicated work!
```

Rewriting this section because it was jumping all over the place